### PR TITLE
Feature/contrast color pie chart

### PR DESCRIPTION
**Pull request by deepaknevdepsl**
### Description
Pie chart contrast color for chart portion, if chart color selected as single color.

### Issues Resolved
Change portion border color for single dark color.

### Check List
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: dashboards-observability/.cypress/integration/1_event_analytics.spec.js [hunk 1]
```diff
@@ -615,4 +616,16 @@ describe('Renders data view', () => {
...
+  beforeEach(() => {
+    landOnEventVisualizations();
+});
+
+it.only('Renders Pie chart with single color', () => {
+  renderPieChart();
+    cy.get('.euiFlexItem.euiFlexItem--flexGrowZero .euiButton__text').eq(2).click();
+    cy.wait(delay);
+  });
 });
```
Comment (abasatwar) [inline dashboards-observability/.cypress/integration/1_event_analytics.spec.js]: add empty line.

File: dashboards-observability/.cypress/utils/event_constants.js [hunk 2]
```diff
@@ -70,4 +74,23 @@ export const landOnPanels = () => {
...
+    cy.get('[name="count()"]').eq(0).click();
+    cy.get('[data-test-subj="comboBoxToggleListButton"]').eq(0).click();
+    cy.get('[data-test-subj="comboBoxInput"]').eq(2).click();
+    cy.get('[name="count()"]').eq(1).click({ multiple: true });
+    cy.get('.euiIEFlexWrapFix').eq(2).contains('Chart Styles').should('exist');
+    cy.get('[data-test-subj="comboBoxInput"]').eq(3).click();
+    cy.get('[name="Pie"]').click();
+    cy.get('.euiSuperSelectControl').click();
+    cy.get('.euiContextMenuItem.euiSuperSelect__item.euiSuperSelect__item--hasDividers').eq(1).click();
 };
```
Comment (abasatwar) [inline dashboards-observability/.cypress/utils/event_constants.js]: same as above.

**Review by abasatwar — CHANGES_REQUESTED** [review]

File: dashboards-observability/.cypress/integration/1_event_analytics.spec.js (hunk 1 above)
Comment (deepaknevdepsl) [reply dashboards-observability/.cypress/integration/1_event_analytics.spec.js]: Added empty line at the end of file.

File: dashboards-observability/.cypress/utils/event_constants.js (hunk 2 above)
Comment (deepaknevdepsl) [reply dashboards-observability/.cypress/utils/event_constants.js]: Added empty line at the end of file.

**Comment by deepaknevdepsl** [comment]
Creating single common PR for pie chart, hence closing this PR.

**Pull request closed without merging**